appeals/web/src/server/appeals/appeal-details/interested-party-comments/view-and-review/view-and-review.controller.js

```diff
@@ -88,3 +98,119 @@ export const postReviewInterestedPartyComment = async (request, response, next)
 		return response.status(500).render('app/500.njk');
 	}
 };
+
+/** @type {import('@pins/express').RequestHandler<Response>} */
+export const getManageFolder = async (request, response) => {
+	const {
+		currentFolder,
+		params: { appealId, commentId }
+	} = request;
+
+	if (!currentFolder) {
+		return response.status(404).render('app/404');
+	}
+
+	const headingTextOverride = 'Supporting Documents';
+	await renderManageFolder({
+		request,
+		response,
+		backLinkUrl: `/appeals-service/appeal-details/${appealId}/interested-party-comments/${commentId}/review`,
+		viewAndEditUrl: `/appeals-service/appeal-details/${appealId}/interested-party-comments/${commentId}/manage-documents/{{folderId}}/{{documentId}}`,
+		pageHeadingTextOverride: headingTextOverride
```

Comment:
If `headingTextOverride` isn't reused you could just pass this as `pageHeadingTextOverride: 'Supporting documents'`